Cache impl_test.go still flaky

Despite #533 
```
ok  	k8s.io/heapster/sinks/api	0.059s	coverage: 72.7% of statements
--- FAIL: TestGC-2 (10.20s)
	Location:	impl_test.go:93
	Error:		Not equal: 7 (expected)
			        != 6 (actual)
		
FAIL
coverage: 92.1% of statements
FAIL	k8s.io/heapster/sinks/cache	15.322s
```

--- a/sinks/cache/impl_test.go
+++ b/sinks/cache/impl_test.go
@@ -71,9 +71,11 @@ func TestGC(t *testing.T) {
 	f.Fuzz(&pods)
 	f.Fuzz(&containers)
 	for i := range pods {
+		pods[i].ID = fmt.Sprintf("ID-%d", i)
 		pods[i].Name = fmt.Sprintf("%d-%s", i, pods[i].Name)
 	}
 	for i := range containers {
+		containers[i].Hostname = fmt.Sprintf("Node-%d", i%5)
 		containers[i].Name = fmt.Sprintf("%d-%s", i, containers[i].Name)
 	}
 

